lib/webpacker/configuration.rb

```diff
@@ -36,6 +36,10 @@ def source
       fetch(:source_path)
     end
 
+    def compile?
+      fetch(:compile).nil? ? Webpacker.env.test? : fetch(:compile)
```

Comment:
I'd drop the `Webpacker.env.test?` default condition and just use the configuration option.